site/content/user-guide/device-registry-http.md

```diff
@@ -0,0 +1,348 @@
++++
+title = "Device Registry - RESTful resources"
+weight = 450
```

Comment:
have you checked if this is in the correct range?

Reply:
Thanks for the comment - yes, it renders it to the very end and leaves a larger gap for possible future page inserts (the setup pages list first from 245 - 390, with unequal distances between the single pages). So I thought 450 might be a reasonably flexible weight.
WDYT?

Reply:
That won't work because the *Developer Guide* pages start at 395 according to `config.toml`

Reply:
Ah thanks, I missed that (as you see ;-))!
So then I will condense the pages in the user guide regarding their weight, so we are clean again.
Will provide an update soon..